pymc3/distributions/multivariate.py

```diff
@@ -2089,3 +2090,127 @@ def logp(self, value):
 
     def _distr_parameters_for_repr(self):
         return ["mu"]
+
+
+class CAR(Continuous):
+    R"""
+    Likelihood for a conditional autoregression. This is a special case of the
+    multivariate normal with an adjacency-structured covariance matrix.
+
+    .. math::
+
+       f(x \mid W, \alpha, \tau) =
+           \frac{|T|^{1/2}}{(2\pi)^{k/2}}
+           \exp\left\{ -\frac{1}{2} (x-\mu)^{\prime} T^{-1} (x-\mu) \right\}
+
+    where :math:`T = (\tau D(I-\alpha W))^{-1}` and :math:`D = diag(\sum_i W_{ij})`.
+
+    ========  ==========================
+    Support   :math:`x \in \mathbb{R}^k`
+    Mean      :math:`\mu \in \mathbb{R}^k`
+    Variance  :math:`(\tau D(I-\alpha W))^{-1}`
+    ========  ==========================
+
+    Parameters
+    ----------
+    mu: array
+        Real-valued mean vector
+    W: Numpy matrix
+        Symmetric adjacency matrix of 1s and 0s indicating
+        adjacency between elements.
+    alpha: float or array
+        Autoregression parameter taking values between -1 and 1. Values closer to 0 indicate weaker
+        correlation and values closer to 1 indicate higher autocorrelation. For most use cases, the
+        support of alpha should be restricted to (0, 1)
+    tau: float or array
+        Positive precision variable controlling the scale of the underlying normal variates.
+    sparse: bool, default=False
+        Determines whether or not sparse computations are used
+
+    References
+    ----------
+    ..  Jin, X., Carlin, B., Banerjee, S.
+        "Generalized Hierarchical Multivariate CAR Models for Areal Data"
+        Biometrics, Vol. 61, No. 4 (Dec., 2005), pp. 950-961
+    """
+
+    def __init__(self, mu, W, alpha, tau, sparse=False, *args, **kwargs):
```

Comment:
Last thing I remembered. Does the distribution ever work without explicitly defining shape? If it does that's fine, otherwise shape should be described in the docstrings (eg as in the Dirichlet Multinomial) and you may want to make it a required named argument as in the Dirichlet Multinomial.

Reply:
Yup - it works without explicitly providing shape information.